src/pybind/mgr/dashboard/requirements.txt

```diff
@@ -1,6 +1,7 @@
 astroid==1.6.1
 attrs==17.4.0
 backports.functools-lru-cache==1.4
+bcrypt==3.1.4
```

Comment:
This seems to be a completely different package. Does this change have any affect on the spec and control files?

Reply:
no, the problem was actually the opposite. The bcrypt package installed in the several distros using the package manager was the `bcrypt` package, and not the `python-bcrypt` package. This change corrects the difference.